### DISCUSSION TITLE
Update parts price from parts search

### DISCUSSION THREAD

**Discussion by einhverfr** [ledgersmb/LedgerSMB#5834] [Ideas]
https://sourceforge.net/p/ledger-smb/feature-requests/305/

**Comment by einhverfr** [comment]
Would need extra test cases, but should be doable in a minor branch without any headache.

**Comment by ehuelsmann** [comment]
@einhverfr the link to SF is now dead. Could you provide some imagery or a longer description of the desired functionality so I can estimate how much work this is?

**Comment by einhverfr** [comment]
The goal would be to go to the parts search results and update prices.    I don't think it is a huge amount of work in the current architecture.   But if we wait for more of a services-oriented architecture, it might be even easier.

I don't think descriptions need to be editable from there.
